libs/api/database/src/lib/notifications/notificationQueue/blog-notification-queue.document.ts

```diff
@@ -2,5 +2,5 @@ import { BlogNotificationInfo } from '@dragonfish/shared/models/notifications/bl
 import { NotificationQueueDocument } from '../notification-queue.schema';
 
 export interface BlogNotificationQueueDocument extends NotificationQueueDocument, BlogNotificationInfo {
-    readonly _id?: string;
+    readonly _id: string;
```

Comment:
not having these caused us compiler errors in the past. make sure this compiles.